presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -203,6 +219,24 @@ public void start()
                     throw t;
                 }
             }, 1, 1, MILLISECONDS);
+            refreshExecutor.scheduleWithFixedDelay(() -> {
```

Comment:
i think we should feature flag this so it won't run in case of resource manager is not enabled.